src/main/java/org/digma/intellij/plugin/toolwindow/common/ThemeChangeListener.java

```diff
@@ -12,6 +12,8 @@
 import static org.digma.intellij.plugin.toolwindow.common.ToolWindowUtil.GLOBAL_SET_UI_THEME;
 import static org.digma.intellij.plugin.toolwindow.common.ToolWindowUtil.REQUEST_MESSAGE_TYPE;
 
+//todo: change implementation because this listener hold a reference to jbCefBrowser
```

Comment:
is it planned for this PR? or in future PR?

Reply:
future